pkg/kubelet/dockertools/fake_docker_client.go

```diff
@@ -206,11 +212,26 @@ func (f *FakeDockerClient) StartContainer(id string, hostConfig *docker.HostConf
 			Config:     &docker.Config{Image: "testimage"},
 			HostConfig: hostConfig,
 			State: docker.State{
-				Running: true,
-				Pid:     os.Getpid(),
+				Running:   true,
+				Pid:       os.Getpid(),
+				StartedAt: time.Now(),
 			},
 			NetworkSettings: &docker.NetworkSettings{IPAddress: "1.2.3.4"},
 		}
+		if f.ContainerMap != nil {
```

Comment:
@fgrzadkowski, first of all, I wasn't trying to block this PR, and I appreciate the test cleanup :-)

I agree it's hard to understand all the tests (who actually does?) in kubelet, but in this case, I think the scope is pretty limited and it was mostly mechanical (e.g., as shown in https://github.com/kubernetes/kubernetes/commit/ba83f5dbf086dc9f86ccd16e95bb0face00fcbda, which passed all unit/integration tests).

Besides the node team, no one owns kubelet, but most people have contributed to the codebase. It'd be great to invest more efforts to not make the situation worse. I didn't know this PR was so urgent, so I apologize if it caused any delay.

> I spent half an hour trying to figure out how it's passed to the kubelet_test and I really don't think it's worth spending more)

@gmarek, kubelet_test doesn't/shouldn't use FakeDockerClient at all. It should simply use container.FakeRuntime and touches no docker-specific bits.